net/shorewall6-lite/patches/120-logfile.patch

```diff
@@ -0,0 +1,13 @@
+Index: shorewall6-lite-5.1.7.2/shorewall6-lite.conf
+===================================================================
+--- shorewall6-lite-5.1.7.2.orig/shorewall6-lite.conf	2017-10-11 15:36:41.037818015 +0200
++++ shorewall6-lite-5.1.7.2/shorewall6-lite.conf	2017-10-11 15:36:41.033818232 +0200
+@@ -28,7 +28,7 @@
+ #			       L O G G I N G
+ ###############################################################################
+ 
+-LOGFILE=
++LOGFILE=/dev/null
```

Comment:
Ditto...

Reply:
Also, does this need to be a patch?  Why not just drop it into `files/etc/` or whatever?

Reply:
I decided to make it a patch because of possible changes of upstream in the config and not to check the conf files for internal changes.